Samples/excel-shared-runtime-scenario/manifest.xml

```diff
@@ -14,14 +14,14 @@
 	<DisplayName DefaultValue="Contoso Data Importer" />
 	<Description DefaultValue="Uses a shared runtime to share global state across all parts of the add-in and provide a responsive ribbon UI."/>
 	<!-- Icon for your add-in. Used on installation screens and the add-ins dialog. -->
-	<IconUrl DefaultValue="https://localhost:3000/assets/icon-32.png" />
-    <HighResolutionIconUrl DefaultValue="https://localhost:3000/assets/icon-80.png"/>
+	<IconUrl DefaultValue="https://davidchesnut.github.io/PnP-OfficeAddins/Samples/excel-shared-runtime-scenario/assets/icon-32.png" />
```

Comment:
Should these be pointing to officedev instead of davidchesnut?

Reply:
Yes, I have to update this after the merge.